lib/pecorino.rb

```diff
@@ -1,8 +1,13 @@
 # frozen_string_literal: true
 
+require "active_support/concern"
+require "active_record/sanitization"
+
 require_relative "pecorino/version"
 require_relative "pecorino/leaky_bucket"
 require_relative "pecorino/throttle"
+require_relative "pecorino/postgres"
+require_relative "pecorino/sqlite"
```

Comment:
**suggestion:** we probably don't need all these adapters at once. should we maybe figure out autoloading or selectively require those?